apps/selvbetjening-ui/src/inntektsjustering/2-inntekt-til-neste-aar/SumAvOppgittInntekt.tsx

```diff
@@ -29,6 +29,9 @@ export const SumAvOppgittInntekt = ({ inntektTilNesteAar }: { inntektTilNesteAar
         if (isNaN(inntektTilNesteAar.inntektFraUtland)) inntekt += 0
         else inntekt += inntektTilNesteAar.inntektFraUtland
 
+        if (!inntektTilNesteAar.AFPInntekt || isNaN(inntektTilNesteAar.AFPInntekt)) inntekt += 0
+        else inntekt += inntektTilNesteAar.AFPInntekt
+
         return `${inntekt} kr`
```

Comment:
Skulle man formatert resultatet her slik at det er mer leselig for bruker? Typ `900.000 kr` fremfor `900000 kr`